uber/templates/registration/checkin.html

```diff
@@ -89,7 +89,14 @@
                 </tr>
                 <tr>
                     <td><strong>Badge Number:</strong></td>
```

Comment:
We were previously asking people ot type in the badge number even though we already have it.  Which is bad since we DO NOT WANT THIS TO CHANGE.  The use of a ``hidden`` form field here is kind of kludgy but it's fine for now.